openlibrary/templates/type/list/view.html

```diff
@@ -111,6 +111,7 @@ <h1>$list.name</h1>
                     \$(seed_id).remove();
                     // update seed count
                     \$("#seed-count").html(render_seed_count(get_seed_count()));
+                    \$("#item_count").load(location.href + " #item_count");
```

Comment:
It looks like this count might do what we want; but there is no `#seed-count` on the page! Can you see if changing this to 

```suggestion
                    \$("#item-count").html(render_seed_count(get_seed_count()));
```

Fixes it? You'll need to also move the `id` to the `<strong>` element so that HTML get's put in the right place.

Reply:
Yes, it fixes the problem :)